cmd/reva/common.go

```diff
@@ -30,8 +30,11 @@ import (
 )
 
 const (
-	viewerPermission string = "viewer"
-	editorPermission string = "editor"
+	viewerPermission  string = "viewer"
```

Comment:
Let's review this @labkode and in case we postpone to another PR. But the `coowner` role is good to have now.